src/pages/trade-types/cfds/_start-trading.tsx

```diff
@@ -53,7 +53,6 @@ const StartTrading = () => {
                         </Timeline.Item>
                     </Timeline>
                     <LinkButton
-                        id="dm-dmt5-signup"
```

Comment:
@niloo-fs, was this causing errors as well?

Reply:
Yes @jim-deriv , but I'll check this with QA so we can make sure that removing this doesn't make any issue in production